wasmplugin/plugin.go

```diff
@@ -87,8 +87,8 @@ func (ctx *corazaPlugin) NewHttpContext(contextID uint32) types.HttpContext {
 		// TODO(jcchavezs): figure out how/when enable/disable metrics
 		metrics: ctx.metrics,
 		logger: ctx.waf.NewTransaction().
-			DebugLogger().
-			With(debuglog.Uint("context_id", uint(contextID))),
+			DebugLogger(), /*.
```

Comment:
- Would we not want the context id in the logs?
- This is good otherwise. We can extend this to include the transaction id with every log.